vortex-btrblocks/src/integer.rs

```diff
@@ -185,13 +186,12 @@ impl Scheme for ConstantScheme {
             return Ok(0.0);
         }
 
-        // Only arrays with one distinct values can be constant compressed.
-        if stats.distinct_values_count != 1 {
+        if stats.src.all_invalid() {
             return Ok(0.0);
         }
```

Comment:
don't we want to use constant compression for all-invalid?

Reply:
a3a8e6747e0282ef3910611419c3a42d811f0e2c